samples/nrf9160/modem_shell/src/gnss/gnss.c

```diff
@@ -829,6 +829,13 @@ static void get_pgps_data_work_fn(struct k_work *work)
 	}
 
 	mosh_print("GNSS: P-GPS response processed");
+
+	err = nrf_cloud_pgps_notify_prediction();
```

Comment:
I am a bit confused now. According to https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/libraries/networking/nrf_cloud_pgps.html calling nrf_cloud_pgps_notify_prediction() is not necessary when using REST which is the case here?

Reply:
Other folks added the REST interface after I did the initial MQTT development of this feature.  I did not catch that the documentation was incorrect.

It is necessary to call nrf_cloud_pgps_notify_prediction() to ensure that newly downloaded predictions are considered, if the reason for the download was that an appropriate prediction was not present.  I verified this in testing, which is why I added it in this PR.

I'll update the documentation.